CONTRIBUTING-HTML.md

```diff
@@ -219,3 +146,198 @@ Once you've finished writing the example, see the [Testing](CONTRIBUTING.md#test
 The final example should look something like this:
 
 ![Final example](https://screenshotscdn.firefoxusercontent.com/images/9e834e75-98a7-4c57-be32-455138aa8c69.png)
+
+## Guidelines
+
+This section describes some guidelines to follow when writing HTML examples. It's split into two sections:
+
+* **Formal guidelines** cover formal aspects of the example, such as how long it should be.
+* **Content guidelines** cover the actual content of the example: that is, what it should include or exclude.
+
+Sometimes formal and content guidelines are at odds. For example, sometimes writing a useful example means making it longer than the formal guidelines ask. Usually, when this happens, we should prioritize content guidelines.
+
+### Formal guidelines
+
+In general: try out your example using `npm run start` and see what it looks like with a browser window width of 1000px.
```

Comment:
Nit: You can also just run `npm start`

Reply:
all these months I've been typing those extra 4 characters! :)